analytic_engine/src/instance/create.rs

```diff
@@ -96,6 +96,11 @@ impl Instance {
             MetaEditRequest {
                 shard_info: TableShardInfo::new(request.shard_id),
                 meta_edit: MetaEdit::Update(meta_update),
+                extr_info: TableCatalogInfo {
```

Comment:
How about refactor field names together, and we merge it after that!

Reply:
ok!